packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -727,7 +727,7 @@ String _getIosEngineArtifactPath(String engineDirectory,
     if (!platformDirectory.basename.startsWith('ios-')) {
       continue;
     }
-    // ios-x86_64-simulator, ios-arm64_x86_64-simulator, ios-armv7_arm64 (Xcode 11), or ios-arm64_armv7 (Xcode 12).
+    // ios-x86_64-simulator, ios-arm64_x86_64-simulator, or ios-arm64.
```

Comment:
From looking at the code, it seems like `_getIosArtifactPath()` will automagically generate the correct path?

Reply:
Not sure what you mean, `_getIosArtifactPath` calls this method `_getIosEngineArtifactPath`
Used like this:
https://github.com/flutter/flutter/blob/e56c83a1ac60c4ebc42e0272d6db89645a034db6/packages/flutter_tools/lib/src/build_system/targets/ios.dart#L300-L305